pages/people/[id]/case-status/[caseStatusId]/edit/edit.tsx

```diff
@@ -34,7 +34,13 @@ const EditCaseStatus = (): React.ReactElement => {
 
   return (
     <>
-      {isScheduledCaseStatus && (
+      {pastCaseStatusStartDate && action == 'edit' && (
```

Comment:
Tiny suggestion to also add `&& pastCaseStatusStartDate !== 'undefined'` to make sure that pastCaseStatusStartDate isn't undefined